src/include/btree_inline.h

```diff
@@ -1723,7 +1723,7 @@ __wt_page_del_visible(WT_SESSION_IMPL *session, WT_PAGE_DELETED *page_del, bool
     }
 
     return (__wt_txn_visible(
-      session, page_del->txnid, page_del->pg_del_start_ts, page_del->pg_del_durable_ts));
+      session, page_del->txnid, get_page_del_start_ts(page_del), page_del->pg_del_durable_ts));
```

Comment:
when about visibility, we should always pass start_ts but we need to check prepared first.